pkg/flow/server.go

```diff
@@ -128,11 +127,7 @@ func (srv *server) start(ctx context.Context) error {
 	}()
 
 	srv.sugar.Debug("Initializing secrets.")
-	srv.secrets, err = initSecrets()
-	if err != nil {
-		return err
-	}
-	defer srv.cleanup(srv.secrets.Close)
+	// TODO: fix secretes.
```

Comment:
This TODO should be resolved before merge. I think it doesn't need to exist?

Reply:
Fixed.